pioneer/packages/joy-media/src/Upload.tsx

```diff
@@ -205,7 +222,7 @@ class Upload extends React.PureComponent<Props, State> {
       });
     } else {
       this.setState({ file, computingHash: true });
-      this.startComputingHash();
+      void this.startComputingHash();
```

Comment:
I don't understand the presence of this `void` here? Is it to drop or ignore the return value of the call ?

Reply:
This is a way to mark a promise which we intentionally don't await. It's required by the new `no-floating-promises` rule (see: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-floating-promises.md#ignorevoid)

I think it's a good rule, because it prevents omitting `await` by accident and marking with `void` also gives a a nice hint that the function we execute is async.